govc/flags/search.go

```diff
@@ -57,6 +57,8 @@ type SearchFlag struct {
 var searchFlagKey = flagKey("search")
 
 func NewSearchFlag(ctx context.Context, t int) (*SearchFlag, context.Context) {
+	searchFlagKey = flagKey(fmt.Sprintf("%s%d", searchFlagKey, t))
```

Comment:
Thanks @makelarisjr . There are a few consumers of govc commands that have their own main program, and may initialize more than once. In which case this would continue to append.  Can you remove the package `var` and change this to:
```go
searchFlagKey := flagKey(fmt.Sprintf("search%d", t))
```

Reply:
Fixed!